bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownButton.html

```diff
@@ -4,15 +4,16 @@
     </head>
     <body>
         <wicket:panel>
-            <a class="btn dropdown-toggle" data-toggle="dropdown" wicket:id="btn" href="#">
-                <span wicket:id="icon"></span>
-                <wicket:container wicket:id="label"></wicket:container>
-            </a>
-            <ul wicket:id="dropdown-menu" class="dropdown-menu">
-                <li wicket:id="buttons">
-                    <a wicket:id="button" class="dropdown-item"></a>
-                </li>
-            </ul>
+           <a class="dropdown-toggle" wicket:id="btn" href="#" type="button" 
```

Comment:
Please either remove `href="#"` or use `href="javascript:;"` instead. Using `"#"` has bad side effect of scrolling the page to the top.

Reply:
Why not use the <button> tag instead ?
https://getbootstrap.com/docs/4.0/components/dropdowns/#single-button-dropdowns

Reply:
Because I want to enable DropUp, DropRight ... variants, However, did not try if this works with the button tag. Its just described with btn-grp.